app/services/profiles/update.rb

```diff
@@ -61,6 +62,7 @@ def update_successful?
 
       Profile.transaction do
         update_profile
+        @profile.update(**@updated_static_profile_attributes)
```

Comment:
Would it make sense to move this into the `update_profile` helper that's called on the line above? Then this helper would include all complexities of updating a profile record.

Reply:
I can actually remove this entirely now that I changed the model. :smile: